<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1524112578036" stop="1524112634539" version="1.4.15" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Default suite : Default test</name>
    <title>Default suite : Default test</title>
    <test-cases>
        <test-case start="1524112579030" stop="1524112632294" status="passed">
            <name>start</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DDC5-D-78KFNK2"/>
                <label name="thread" value="8788@DDC5-D-78KFNK2.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1524112634499" stop="1524112634502" status="broken">
            <name>printResults</name>
            <description>@AfterTest</description>
            <failure>
                <message>NoSuchMethodError: org.apache.poi.xssf.usermodel.XSSFCellStyle.setFillPattern(S)V</message>
                <stack-trace>java.lang.NoSuchMethodError: org.apache.poi.xssf.usermodel.XSSFCellStyle.setFillPattern(S)V
	at com.test.sample.project.seleniumautomation.ProductEnrichmentValidationWithMethods.printResults(ProductEnrichmentValidationWithMethods.java:257)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.TestRunner.afterRun(TestRunner.java:958)
	at org.testng.TestRunner.run(TestRunner.java:606)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DDC5-D-78KFNK2"/>
                <label name="thread" value="8788@DDC5-D-78KFNK2.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
